### ISSUE TITLE
Floating point exception (core dumped)

### ISSUE THREAD

**Issue by xuezhongfei2008** [rhasspy/piper#7]
turn onnx

**Comment by synesthesiam** [comment]
Can you provide more information please?

**Comment by xuezhongfei2008** [comment]
hello, 
my environment:
	Linux ubuntu516 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
	Python 						3.8.13
	torch                        1.12.1
	torchaudio                   0.12.1
	torchinfo                    1.7.1
	torchmetrics                 0.9.0
	torchvision                  0.13.1
	cudatoolkit               	11.3.1               	
	onnxruntime                  1.13.1

when use_sdp=False   G_**.pth convert to onnx  ,is ok
when use_sdp=True   G_**.pth convert to onnx   Floating point exception (core dumped)

**Comment by xuezhongfei2008** [comment]
> Can you provide more information please?


hello, 
my environment:
	Linux ubuntu516 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
	Python 						3.8.13
	torch                        1.12.1
	torchaudio                   0.12.1
	torchinfo                    1.7.1
	torchmetrics                 0.9.0
	torchvision                  0.13.1
	cudatoolkit               	11.3.1               	
	onnxruntime                  1.13.1

when use_sdp=False   G_**.pth convert to onnx  ,is ok
when use_sdp=True   G_**.pth convert to onnx   Floating point exception (core dumped)

**Comment by synesthesiam** [comment]
The model itself comes from here: https://github.com/jaywalnut310/vits/
I've never had this problem when exporting a model with `use_sdp=True`. I would either try a different version of PyTorch or lower `OPSET_VERSION` in `export_onnx.py`

**Comment by tuannvhust** [comment]
@xuezhongfei2008 i got the same issue, did you solve this bug?

**Comment by erikli** [comment]
Although there are no details in this issue, I believe I experienced the same bug while trying to run `python -m larynx_train.export_onnyx`. For my own case, I was able to fix it by editing `requirements.txt` to use newer version of torch and onnxruntime. For reference, I am on Python 3.10.9, using Arch Linux through WSL.


requirements.txt:
```
cython>=0.29.0,<1
espeak-phonemizer>=1.1.0,<2
librosa>=0.9.2,<1
numpy>=1.19.0
onnxruntime~=1.13.0
pytorch-lightning~=1.7.0
torch~=1.13.0
```

Then I just activate `venv` again and then `pip3 install -r requirements.txt`.

Does this fix things for others?

**Comment by d-r-a-b** [comment]
Although there are no details in this issue, I believe I experienced the same bug while trying to run `python -m larynx_train.export_onnyx`. For my own case, I was able to fix it by editing `requirements.txt` to use newer version of torch and onnxruntime. For reference, I am on Python 3.10.9, using Arch Linux through WSL.


requirements.txt:
```
cython>=0.29.0,<1
espeak-phonemizer>=1.1.0,<2
librosa>=0.9.2,<1
numpy>=1.19.0
onnxruntime~=1.13.0
pytorch-lightning~=1.7.0
torch~=1.13.0
```

Then I just activate `venv` again and then `pip3 install -r requirements.txt`.

Does this fix things for others?
